src/proc/color-formats-converter.h

```diff
@@ -61,4 +61,30 @@ namespace librealsense
             color_converter(name, RS2_FORMAT_RGB8, RS2_STREAM_INFRARED) {};
         void process_function(byte * const dest[], const byte * source, int width, int height, int actual_size, int input_size) override;
     };
+
+    class Y411_converter : public functional_processing_block
```

Comment:
Why the mix of upper and lower-case? I don't see the other classes in this file doing this...

Reply:
done